test/spec/modules/cpmstarBidAdapter_spec.js

```diff
@@ -0,0 +1,176 @@
+import { expect } from 'chai';
+import { spec } from 'modules/cpmstarBidAdapter';
+import { deepClone } from 'src/utils';
+
+describe('Cpmstar Bid Adapter', function () {
+  describe('isBidRequestValid', function () {
+    it('should return true since the bid is valid',
+      function () {
+        var bid = { params: { placementId: 123456 } };
+        expect(spec.isBidRequestValid(bid)).to.equal(true);
+      })
+
+    it('should return false since the bid is invalid', function () {
+      var bid = { params: { placementId: '' } };
+      expect(spec.isBidRequestValid(bid)).to.equal(false);
+    })
+
+    it('should return a valid player size', function() {
+      var bid = { mediaTypes: {
+        video: {
+          playerSize: [[960, 540]]
+        }
+      }}
+      expect(spec.getPlayerSize(bid)[0]).to.equal(960);
+      expect(spec.getPlayerSize(bid)[1]).to.equal(540);
+    })
+
+    it('should return a default player size', function() {
+      var bid = { mediaTypes: {
+        video: {
+          playerSize: null
+        }
+      }}
+      expect(spec.getPlayerSize(bid)[0]).to.equal(640);
+      expect(spec.getPlayerSize(bid)[1]).to.equal(440);
+    })
+  });
+
+  describe('buildRequests', function () {
+    const valid_bid_requests = [{
+      'bidder': 'cpmstar',
+      'params': {
+        'placementId': '57'
+      },
+      'sizes': [[300, 250]],
+      'bidId': 'bidId'
+    }];
+
+    const bidderRequest = {
+      refererInfo: {
+        referer: 'referer',
+        reachedTop: false,
+      }
+
+    };
+
+    it('should produce a valid production request', function () {
+      var requests = spec.buildRequests(valid_bid_requests, bidderRequest);
+      expect(requests[0]).to.have.property('method');
+      expect(requests[0]).to.have.property('url');
+      expect(requests[0]).to.have.property('bidRequest');
+      expect(requests[0].url.startsWith('//server.cpmstar.com/view.aspx')).to.equal(true);
```

Comment:
The `startsWith` property is not compatible with IE browsers - which is why the browserstack tests are failing for this PR.  Can you use an alternate assertion for this part of the tests?

I think you could use chai's `includes` instead to get the same effect (see example below).
`expect(requests[0].url).to.include('//dev.server.cpmstar.com/view.aspx');`

Reply:
This test has been updated for IE compatibility.